packages/compose-tunnel-agent/src/ssh/tunnel-set.ts

```diff
@@ -0,0 +1,50 @@
+import net from 'net'
+import { RunningService } from '../docker'
+
+interface Forward {
```

Comment:
prefer type to interface

Reply:
we had this in a previous PR.
there is no significant difference, unless you need specific features, and in that case if u have to choose, the typescript team actually recommends interface.